content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,148 @@
+     model common name by clicking the :guilabel:`expand` button.
+
+     The model technical name can also be accessed in :ref:`developer mode <developer-mode>`. On a
+     form navigate to any field, and hover over the field name. A box of backend information reveals
+     itself with the specific Odoo :guilabel:`Object` name in the backend. This is the technical
+     name of the model that should be added.
+
+- :guilabel:`Record Rules`: lists the second layer of editing and visibility rights. Record Rules
+  overwrite, or refine, the group's *Access Rights*. For each rule, choose values for the following
+  options:
+
+  - :guilabel:`Apply for Read`.
+  - :guilabel:`Apply for Write`.
+  - :guilabel:`Apply for Create`.
+  - :guilabel:`Apply for Delete`.
+
+  .. important::
+     Record rules are written using a *Domain*, or conditions that filter data. A domain expression
+     is a list of such conditions. Users without knowledge of domains and domain expressions should
+     consult an Odoo Business Analyst or the Odoo Support Team before making changes.
+
+Superuser mode
+==============
+
+*Superuser mode* allows the user to bypass record rules and access rights. To activate *Superuser
+mode*, first, activate :ref:`developer mode <developer-mode>`. Then navigate to the :guilabel:`🪲
+(debug)` menu in the top banner. The :guilabel:`🪲 (debug)` menu can be accessed by clicking on the
+:guilabel:`🪲 (debug)` icon. Finally, towards the bottom click :guilabel:`Become Superuser`.
 
 .. important::
-   Making changes in access rights can have a big impact on the database. For this reason, we
-   recommend you to contact your Odoo Business Analyst or our Support Team, unless you have
-   knowledge about Domains in Odoo.
+   Only users with :guilabel:`Settings` access for the Administration section of the *Access Rights*
+   are allowed to log in to *Superuser mode*.
+
+.. danger::
+   *Superuser mode* allows for circumvention of record rules and access rights, and therefore should
+   be exercised with extreme caution. Upon exiting *Superuser mode*, users may be locked out of the
+   database due to changes that were made. This can cause *impotent admin*, or an administrator
+   without the ability to change access rights/settings. In this case contact Odoo Support here:
+   `new help ticket <https://www.odoo.com/help>`_. The support team will be able to restore access
+   using a support login.
+
+To leave *Superuser mode*, log out of the account by navigating to the upper right corner and
+clicking on the :guilabel:`OdooBot` username, then click :guilabel:`Log out`.
+
+.. tip::
+   An alternative to using the :guilabel:`🪲 (debug)` menu to activate *Superuser mode* is to login
+   as a superuser. To do so navigate to the login screen and enter the appropriate :guilabel:`Email`
+   and :guilabel:`Password`. Instead of clicking :guilabel:`Login`, click :guilabel:`Log in as
+   superuser`.
```

Comment:
```suggestion
   An alternative way to activate *Superuser mode* is to login as a superuser. To do that, navigate
   to the login screen, and enter the appropriate :guilabel:`Email` and :guilabel:`Password`.
   
   Instead of clicking :guilabel:`Login`, click :guilabel:`Log in as superuser`.
```